### PR TITLE
Displays DSCP and burst size as 0 for undefined values

### PR DESCRIPTION
**Pull request by vishal-j**

### CODE REVIEW

File: app/network_policies/bandwidthpolicydetailsctrl.ts [hunk 1]
```diff
@@ -36,6 +36,8 @@ export class BandwidthPolicyDetailsComponent {
         /* Get particular Profile for based on key*/
         bandwidthPolicyDetailsCtrl.netprofilesModel.getModelByKey(activatedRoute.snapshot.params['key'],false,undefined)
             .then(function (policy) {
+                if (policy.DSCP === undefined) policy.DSCP = 0;
```
Comment (dseevr) [inline app/network_policies/bandwidthpolicydetailsctrl.ts]: How are policy.DSCP and policy.burst undefined at this point? Shouldn't they be null after parsing the JSON response?

Comment (vishal-j) [inline app/network_policies/bandwidthpolicydetailsctrl.ts]: If you set them as 0, netmaster doesn't save these fields in netprofile object and are not present when you do a GET.

Comment (dseevr) [inline app/network_policies/bandwidthpolicydetailsctrl.ts]: Okay, I see that they are `omitempty` there... thanks

**Comment by vishal-j** [comment]
@dseevr @shampur I made some changes after talking to Bill and Joji. DSCP and burst are not mandatory fields. Please review.

File: app/network_policies/bandwidthpolicycreatectrl.ts [hunk 2]
```diff
@@ -79,6 +79,12 @@ export class BandwidthPolicyCreateComponent implements OnInit {
             bandwidthPolicyCreateCtrl.newPolicy.bandwidth = bandwidthPolicyCreateCtrl.newPolicy.bandwidthNumber
                 + " "+ bandwidthPolicyCreateCtrl.newPolicy.bandwidthUnit;
 
+            if (bandwidthPolicyCreateCtrl.newPolicy.DSCP == null) {//burst is null or undefined
```
Comment (dseevr) [inline app/network_policies/bandwidthpolicycreatectrl.ts]: Comment should say "DSCP" 😛

File: app/network_policies/bandwidthpolicydetailsctrl.ts [hunk 3]
```diff
@@ -88,6 +89,13 @@ export class BandwidthPolicyDetailsComponent {
         if (validform) {
             bandwidthPolicyDetailsCtrl.crudHelperService.startLoader(bandwidthPolicyDetailsCtrl);
             bandwidthPolicyDetailsCtrl.policy.bandwidth = bandwidthPolicyDetailsCtrl.policy.bandwidthNumber + " " + bandwidthPolicyDetailsCtrl.policy.bandwidthUnit;
+
+            if (bandwidthPolicyDetailsCtrl.policy.DSCP == null) {//burst is null or undefined
```
Comment (dseevr) [inline app/network_policies/bandwidthpolicydetailsctrl.ts]: here too

**Pull request merged**